OTEL: Dynamic service name for tracing in K8s environment (#32140)
keycloak/keycloak: docs/guides/server/tracing.adoc

Proper HTTP headers (especially `tracestate`) filtering and adequate measures of caller trust would need to be assessed.

For more information, see the https://www.w3.org/TR/trace-context/#security-considerations[W3C Trace context] document.

== Tracing in Kubernetes environment
When the tracing is enabled when using the {project_name} Operator, certain information about the deployment is propagated to the underlying containers.

> NOTE: There is no support for tracing configuration in {project_name} CR yet, so the `additionalValues` can be used for demonstrating purposes by adding the `tracing-enabled` property.

Reviewer: `additionalOptions` is a supported way to configure Keycloak
```suggestion
NOTE: There is no support for tracing configuration in {project_name} CR yet, so the `additionalOptions` can be used to the `tracing-enabled` property and other tracing options.
```
